### PR TITLE
Sortby date

### PR DESCRIPTION
**Pull request by narharikale**
- added date 
- added sort by date

### CODE REVIEW

File: src/frontend/screens/filterScreen/FilterScreen.jsx [hunk 1]
```diff
@@ -18,19 +18,36 @@ function Filter() {
...
         return data ;
+    }   
     
-    }    
+    function sortByDate(state , data) {
+        if(state.SortByDate && state.SortByDate ==='oldest'){
+            return [...data].sort(
+                
+                 (a, b) => {
+                     console.log(a, b ,"a-b")
```
Comment (prathmeshjagtap) [inline src/frontend/screens/filterScreen/FilterScreen.jsx]: remove console.log

File: src/frontend/screens/filterScreen/FilterScreen.jsx [hunk 2]
```diff
@@ -18,19 +18,36 @@ function Filter() {
...
+        }
+        return data
+    }  
 
     const { notesData } = useNotes();
     const { filterState }  = useFilter()
     const filteredData =  filterByPriority(filterState , notesData )
-
+    const finalData = sortByDate( filterState, filteredData )
+    console.log(finalData , "final")
```
Comment (prathmeshjagtap) [inline src/frontend/screens/filterScreen/FilterScreen.jsx]: same here

**Review by prathmeshjagtap — APPROVED** [review]
Rest everything is good

**Review by MiheerTamkhane — APPROVED** [review]
The Code is nicely written and working smooth.
Suggession : make Landing page more attractive.

File: src/frontend/screens/filterScreen/FilterScreen.jsx [hunk 3]
```diff
@@ -1,46 +1,54 @@
...
 function Filter() {
+  function filterByPriority(state, data) {
+    if (state.FilterByPriority && state.FilterByPriority === "high") {
+      return data.filter((note) => note.priority === "High");
+    }
+    if (state.FilterByPriority && state.FilterByPriority === "medium") {
+      return data.filter((note) => note.priority === "Medium");
+    }
+    if (state.FilterByPriority && state.FilterByPriority === "low") {
+      return data.filter((note) => note.priority === "Low");
```
Comment (t007rushi) [inline src/frontend/screens/filterScreen/FilterScreen.jsx:15]: you can directly filter out matching in single line as
```state.FilterByPriority ===note.priority```

**Review by t007rushi — APPROVED** [review]
check for suggestion

**Review by JowelTisso — APPROVED** [review]

**Pull request merged**